_episodes/01-basics.md

```diff
@@ -28,41 +28,40 @@ Word's [Track Changes](https://support.office.com/en-us/article/Track-changes-in
 
 Version control systems start with a base version of the document and
-then save just the changes you made at each step of the way. You can
-think of it as a tape: if you rewind the tape and start at the base
-document, then you can play back each change and end up with your
-latest version.
+then record changes you make each step of the way. You can
+think of it as a recording of your progress: you can rewind to start at the base
+document, play back each change you made, eventually arriving at your
+more recent version.
 
 ![Changes Are Saved Sequentially](../fig/play-changes.svg)
 
 Once you think of changes as separate from the document itself, you
-can then think about "playing back" different sets of changes onto the
-base document and getting different versions of the document. For
-example, two users can make independent sets of changes based on the
-same document.
+can then think about "playing back" different sets of changes, resulting in 
+different versions of the document. For example, two users can make independent
+sets of changes on the same document. 
 
 ![Different Versions Can be Saved](../fig/versions.svg)
 
-Unless there are conflicts, you can even play two sets of changes onto the same base document.
+Unless there are conflicts, you can even incorporate two sets of changes into the same base document.
 
 ![Multiple Versions Can be Merged](../fig/merge.svg)
 
-A version control system is a tool that keeps track of these changes for us and
-helps us version and merge our files. It allows you to
-decide which changes make up the next version, called a
-[commit]({{ page.root }}/reference/#commit), and keeps useful metadata about them. The
+A version control system is a tool that keeps track of these changes for us,
+effectively creating different versions of our files. It allows us to
+decide which changes will be made to the next version (each record of these changes is called a
+[commit]({{ page.root }}/reference/#commit)), and keeps useful metadata about them. The
 complete history of commits for a particular project and their metadata make up
 a [repository]({{ page.root }}/reference/#repository). Repositories can be kept in sync
-across different computers facilitating collaboration among different people.
+across different computers, facilitating collaboration among different people.
 
 > ## The Long History of Version Control Systems
 >
 > Automated version control systems are nothing new.
 > Tools like RCS, CVS, or Subversion have been around since the early 1980s and are used by many large companies.
-> However, many of these are now becoming considered as legacy systems due to various limitations in their capabilities.
-> In particular, the more modern systems, such as Git and [Mercurial](https://swcarpentry.github.io/hg-novice/)
+> However, many of these are now considered legacy systems (i.e., outdated) due to various limitations in their capabilities.
```

Comment:
Definitely think adding this definition is a great idea. Not all our learners may know this term, particularly those who learned English as a second language. :thumbsup:

Reply:
👍

Reply:
Obligatory emoji 👍